README.md

```diff
@@ -130,6 +130,7 @@ Chainloop supports the collection of the following pieces of evidence types:
 - [OpenVEX](https://github.com/openvex)
 - [SARIF](https://docs.oasis-open.org/sarif/sarif/v2.1.0/)
 - [JUnit](https://www.ibm.com/docs/en/developer-for-zos/14.1?topic=formats-junit-xml-format)
+- [Helm Charts](https://helm.sh/docs/topics/charts/)
```

Comment:
also, if you want to add the documentation in this PR. I am ok if you do it in another one, you'll need to also update https://github.com/chainloop-dev/chainloop/blob/main/docs/docs/reference/operator/contract.mdx

That's the source code of the doc site. So probably let's do it in another PR